lesson_28/tests/test_user_registration.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+
+def test_user_registration(driver, open_main_page, start_registration, fill_and_submit_registration_form,
+                           redirection_to_my_profile):
```

Comment:
This test will only work for the first run
On other runs this user will be already registered and test will fail
Use unique data each run